modules/microshift-manifests-removal-scenario.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * microshift//running_applications/microshift-deleting-resource-manifests.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-manifests-removal-scenario_{context}"]
+== Manifest removal scenario
+Use the following procedure in the data removal scenario to completely delete the resource defined in the kustomize manifests.
+
+.Procedure
+
+. Identify the manifest that needs to be placed in the `delete` subdirectories.
+. Use one of the following steps to create the `delete` subdirectories where the manifest is placed for deletion:
+* For RPM installation, create the `delete` subdirectory by running the following command:
++
+[source,terminal]
+----
+sudo mkdir -p _<path_of_deletedir>_ <1>
+----
+<1> Replace `_<path_of_deletedir>_` with the path of delete subdirectory such as `/etc/microshift/manifests.d/delete` or `/etc/microshift/manifests/delete/`.
+* For OSTree deployments, create the `delete` subdirectory by running the following command:
++
+[source,terminal]
+----
+sudo mkdir -p _<path_of_deletedir>_ <1>
+----
+<1> Replace `_<path_of_deletedir>_` with the path of delete subdirectory such as `/usr/lib/microshift/manifests.d/delete/*` or `/usr/lib/microshift/manifests/delete/`.
++
+[IMPORTANT]
+====
+For OSTree installation, the `delete` subdirectories are read only. Package the manifest into an RPM, then add them to the blueprint file before moving the manifest into the `delete` subdirectories.
```

Comment:
I don't think we can write 
> add them to the blueprint file before moving the manifest into the `delete` subdirectories

Packaging them doesn't let us write to /usr.

```suggestion
For OSTree installation, the `delete` subdirectories are read only. Package the manifest into an RPM, then add them to the blueprint file to install it into correct location.
```

Reply:
updated and included in the OSTree module